src/presets/dev/src/dev-print.scss

```diff
@@ -29,7 +29,7 @@
 @use '@ecl/vanilla-component-expandable/expandable-print';
 @use '@ecl/vanilla-component-fact-figures/fact-figures-print';
 @use '@ecl/vanilla-component-file/file-print';
-@use '@ecl/vanilla-component-footer-core/footer-core-print';
+// @use '@ecl/vanilla-component-footer-core/footer-core-print';
```

Comment:
Having one and the same contents for both cases, I think it's save to keep this line.

Reply:
Double-checked, still relevant